app/core/constants.js

```diff
@@ -36,6 +36,7 @@ export const DATA_DONATION_NONPROFITS = [
   { value: 'JDRF', label: 'JDRF' },
   { value: 'NSF', label: 'Nightscout Foundation' },
   { value: 'T1DX', label: 'T1D Exchange' },
+  { value: 'AADE', label: 'American Association of Diabetes Educators' },
```

Comment:
I see that the trello card description shows the name as `AADE Foundation`, which I imagine is likely what was intended to be used as the label.